go.mod

```diff
@@ -7,9 +7,7 @@ require (
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.0-20210507181900-4e0be8d2fbb4
 	github.com/drand/kyber v1.1.4
 	github.com/drand/kyber-bls12381 v0.2.1
-	github.com/filecoin-project/go-address v0.0.5
-	github.com/filecoin-project/go-amt-ipld/v2 v2.1.1-0.20201006184820-924ee87a1349 // indirect
-	github.com/filecoin-project/go-state-types v0.1.1-0.20210915140513-d354ccf10379
-	github.com/filecoin-project/lotus v1.13.0
```

Comment:
Cool, I never liked having this dependency in this library. Looks like github.com/filecoin-project/go-state-types is rather new-ish? That's useful.